kolibri/plugins/facility/assets/src/views/ManageClassPage/index.vue

```diff
@@ -92,6 +101,72 @@
         @cancel="closeModal"
         @success="handleCreateSuccess()"
       />
+
+      <ClassRenameModal
+        v-show="openRenameModal"
+        :classname="classDetails.name"
+        :classid="classDetails.id"
+        :classes="classes"
+        @cancel="closeModal"
+        @success="handleRenameSuccess()"
+      />
+      <SidePanelModal
+        v-show="openCopyClassPanel"
+        ref="resourcePanel"
+        alignment="right"
+        sidePanelWidth="700px"
+        closeButtonIconType="close"
+        @closePanel="openCopyClassPanel = false"
+      >
+        <template #header>
+          <h1 class="side-panel-title">{{ copyClasslabel$() }}</h1>
+        </template>
+        <template #default>
+          <div>
+            <KTextbox
+              v-model="copiedClassName"
+              type="text"
+              :label="classTitleLabel$()"
+              :autofocus="true"
+              :maxlength="120"
+              :showInvalidText="true"
+              :invalid="isClassNameInvalid"
+              :invalidText="classNameAlreadyExists$()"
+            />
+
+            <p class="side-panel-subtitle">{{ coachesAssignedToClassLabel$() }}</p>
+            <SelectableList
+              :value="classCoachesIds"
+              :options="classCoaches"
+              :ariaLabelledby="coachesAssignedToClassLabel$()"
```

Comment:
AriaLabelledBy receives the id of a DOM node, not the label itself, so what we can do instead is to add an id to the `<p>` element we have above, where we have to render the "Coaches assigned to this class" text, and use the same id here.

Reply:
Great catch!